file-formats/fugr/func.json

```diff
@@ -26,32 +29,32 @@
     },
     "processingType": {
       "title": "Processing Type",
-      "description": "Processing Type",
+      "description": "Processing type",
       "type": "string",
       "enum": [
         "normal",
         "rfc",
         "update"
       ],
       "enumDescriptions": [
-        "Normal",
-        "RFC",
-        "Update"
+        "normal",
+        "rfc",
+        "update"
       ]
     },
     "rfcProperties": {
-      "title": "RFC specific fields",
+      "title": "RFC Specific Fields",
       "description": "Specific fields for rfc function modules",
       "type": "object",
       "properties": {
-        "basXmlEnabled": {
-          "title": "basXml enabled",
-          "description": "When this indicator is set, the developer of the function module shows that the function module for the classic RFC protocol and the basXML protocol behaves in the same way, from a semantic point of view",
+        "basxmlEnabled": {
+          "title": "BasXml Enabled",
+          "description": "Function module for classic RFC protocol and basXML protocol behaves the same from a semantic point of view",
```

Comment:
I do not see the value added by this explanation above. 
The full explanation would be the:
When this indicator is set, the developer of the function module shows that the function module for the classic RFC protocol and the basXML protocol behaves in the same way, from a semantic point of view. Semantic differerences result from the conceptional differences between the two protocols. The classic RFC protocol uses a position logic, while the basXML protocol uses a name identification.

Reply:
So I change the description to what is was before? I think the long text could be too long

Reply:
When this indicator is set, it shows that the function module for the classic RFC protocol and the basXML protocol behaves in the same way, from a semantic point of view. The classic RFC protocol uses a position logic, while the basXML protocol uses a name identification.

Reply:
Ok, thank you!